vuln_checker/load.py

```diff
@@ -112,8 +130,8 @@ def to_unified_model(self, model: dict) -> VulnerabilityModel:
         return VulnerabilityModel(
             id=model["name"],
             description=",".join(model["details"]),
-            severity=model["threat_severity"],
-            cvss_v3_score=cvss3.get("cvss3_base_score"),
+            severity=normalize_severity(model.get("threat_severity", "")),
+            cvss_v3_score=safe_str_to_float(cvss3.get("cvss3_base_score")),
```

Comment:
there is no default value, right?
or `cvss3_base_score` always exists...

Reply:
@afdesk score is not always there, which value should be used by default?

Reply:
let's set 0.0